bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -23,6 +23,7 @@
 import com.google.common.collect.Lists;
 import com.google.common.util.concurrent.ListenableFuture;
 import io.netty.buffer.ByteBuf;
+
```

Comment:
Nit: any reason for a new line here?

Reply:
probably came in in cherry-pick. will remove.